### ISSUE TITLE
What about adding a validation against null or white space?

### ISSUE THREAD

**Issue by arivera12** [JeremySkinner/FluentValidation#1174]
I know not empty just validate against empty string but not against white space strings. How about adding this extension method? I know must of the cases we are working on systems we don't want users to put white space strings on the inputs cause it doesn't makes any sense at all. We still need to validate the user as entered something rather different than white space.
```
public static IRuleBuilder<T, TProperty> NotNullOrWhiteSpace<T, TProperty>(this IRuleBuilder<T, TProperty> ruleBuilder)
{
      return ruleBuilder.Must(x => !string.IsNullOrWhiteSpace(x?.ToString()));
}
```

**Comment by arivera12** [comment]
I opened this this issues has recommendation cause in the method comments didn't said it validates white space strings and also found people asking for this. I read the source code and I found this and its seems to be implemented.
![image](https://user-images.githubusercontent.com/5600102/60643048-d9ee3880-9dff-11e9-9b90-af113ca56635.png)
I did a quick test and works as expected. My bad for opening this issue. Regards.
